roles/aws/admin-route53/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# defaults file for admin-route53
+aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+zone_add: []
```

Comment:
Can we please align this with the content of `roles/config-dns-server`. In that way, we can easily switch back/forth between Route53 and  bind/named. For example: In the `config-dns-server` role, this is called `named_config_views` ... what if we update that to be `dns_server_views` and hence re-use the same structure for both.

Reply:
@oybed I can't see the point of doing this. Roles 'config-dns-server' and 'admin-route53' have different purposes. I see the role 'admin-route53' more aligned to the 'dns' role rather than 'config-dns-server'.

If we rename 'zone_add' we will still need another key or var to understand if the zone or records from the zone must be present or not.